tools/mitohifi/mitohifi.xml

```diff
@@ -0,0 +1,349 @@
+<tool id='mitohifi' name='MitoHiFi' version='@TOOL_VERSION@+@GALAXY_TOOL_VERSION@' profile='20.01'>
+    <description>assembly mitogenomes from Pacbio HiFi reads</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro='edam_ontology' />
+    <expand macro='requirements' />
+    <version_command>python mitohifi_v2.py -h</version_command>
+    <command detect_errors='exit_code'><![CDATA[
+        #if $operation_mode.command == 'find_reference'
+            findMitoReference.py 
+            --species '${operation_mode.species}'
+            --email '${operation_mode.email}'
+            --outfolder ./ 
+            #if $operation_mode.min_length
+                --min_length $operation_mode.min_length
+            #end if
+            $operation_mode.exact_specie
+            && mv ./*fasta ./reference_genome.fasta
+            && mv ./*gb ./reference_genome.gb
+        #elif $operation_mode.command == 'mitohifi'
+            mitohifi_v2.py 
+            #if $operation_mode.input_option.input == 'pacbio'
+                -r '${ ' '.join(["%s" % $x for $x in $operation_mode.input_option.input_reads]) }'
```

Comment:
single-quotes, I think you generate here double-quotes

Reply:
It is fine I think, this is the output command that it generates `mitohifi_v2.py -r '/tmp/tmp_brg1r0y/files/000/dataset_91.dat /tmp/tmp_brg1r0y/files/000/dataset_92.dat' -f '/tmp/tmp_brg1r0y/files/000/dataset_3.dat' -g '/tmp/tmp_brg1r0y/files/000/dataset_2.dat' -p 50 -o 5 -t ${GALAXY_SLOTS:-1} `